src/Providers/Mistral/Client.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EchoLabs\Prism\Providers\Mistral;
+
+use Illuminate\Http\Client\PendingRequest;
+use Illuminate\Http\Client\Response;
+use Illuminate\Support\Facades\Http;
+
+class Client
+{
+    protected PendingRequest $client;
+
+    public function __construct(
+        public readonly string $url,
+        public readonly ?string $apiKey,
+    ) {
+        $this->client = Http::withHeaders(array_filter([
+            'Authorization' => $this->apiKey ? sprintf('Bearer %s', $this->apiKey) : null,
+        ]))->baseUrl($this->url);
```

Comment:
Since the API key is required we can probably remove the conditional here.

Reply:
Pushed the changes. [9802f42](https://github.com/echolabsdev/prism/pull/13/commits/9802f423c1b91da29b93098baab58e8e505dd8b5)